### DISCUSSION TITLE
read carrier sense inside RedMatic

### DISCUSSION THREAD

**Discussion by OHacks** [rdmtc/RedMatic#591] [General]
I am looking for a way to read carrier sense inside RedMatic.
How do I do that?

**Comment by Tscherno** [comment]
You mean the duty cycle? There is a variable defined in the CCU DutyCycle which can be read easily. At least for raspberrymatic it is there.

**Comment by OHacks** [comment]
No, not the duty cycle, the carrier sense is (more or less) new value that shows if there are any disruptive signals.

**Comment by Tscherno** [comment]
OK i see. Doesn't seem to be populated. You could add a change request to rasperrymatic to let it add to a variable as well. Or you can try webscraping with nodered. HTTP-request and parsing the response for #carrierSenseVal0

**Comment by hobbyquaker** [comment]
Der Carrier-Sense-Wert wird von der CCU nicht als Datenpunkt bereitgestellt (siehe Thread). Geschlossen.

---
*Written by Claude Fable on behalf of hobbyquaker.*
